inspire_matcher/validators.py

```diff
@@ -109,3 +109,10 @@ def persistent_identifier_validator(record, result):
     result_pid_values = {frozenset(pid.values()) for pid in result_pids}
 
     return bool(record_pid_values & result_pid_values)
+
+
+def arxiv_eprints_validator(record, result):
+    record_eprints = get_value(record, "arxiv_eprints.value", [])
+    result_eprints = get_value(result, "_source.arxiv_eprints.value", [])
+
+    return bool(record_eprints & result_eprints)
```

Comment:
this line seems to be incorrect cause if there is an arxiv you don't want to do a match, so it should be False in that case.